collections/St_Johns_College/SW_V1_St_Johns_College_MS_007.xml

```diff
@@ -0,0 +1,228 @@
+                        and text ink.</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="rubrication">Headings in red.</decoNote>
+                     <decoNote type="flourInit">At book heads, 6-line red and blue lombards with
+                        flourishing of the same, including marginal extensions (for Genesis, the
+                        flourish and sprays are extended into a demivinet).</decoNote>
+                     <decoNote type="flourInit">Less ornate 3-line blue lombards on red flourishing
+                        for chapters, their numbers within the column in red.</decoNote>
+                     <decoNote type="paraph">Some chapters broken by red and blue paraphs (following
+                        rubricated instructions for them in margin ‘/ /’).</decoNote>
+                     <decoNote type="paratext">Running titles in red preceded by a blue paraph give
+                        book title and chapter number.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <p>A modern replacement. Sewn on six thongs. At the front and rear, single
+                        marbled paper flyleaves (the rear ii).</p>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notBefore="1400" notAfter="1425">s. xv
+                        in.</origDate>
+                     <origPlace>
+                        <country key="place_7002445">England</country>
+                     </origPlace>
+                  </origin>
+                  <provenance>‘<persName key="person_4619" role="sgn">Robert</persName>’ (<locus
+                        from="47r" to="47r">fol. 47</locus>, lower margin); more fully, ‘<persName
+                        key="person_4619" role="sgn">Robert Huys</persName>’ (<locus from="79r"
+                        to="79r">fol. 79</locus>, in the same position, <date calendar="Gregorian"
+                        notBefore="1490" notAfter="1510">s. xv/xvi</date>).</provenance>
+                  <provenance>‘The translation of the Bible in Englishe by <persName
+                        key="person_68939042" role="trl">Master Iohn Wicklife</persName> in the time
+                     of King Edward the third <note type="editorial">&lt;a short further line
+                        erased&gt;</note>' (<date calendar="Gregorian" notBefore="1600"
+                        notAfter="1700">s. xvii</date>, <locus from="1r" to="1r">fol.
+                     1</locus>).</provenance>
+                  <acquisition when="1620" xml:lang="la">‘Liber Collegij Sanctj Iohannis Baptistae
+                     Oxon’ ex dono <persName key="person_4620" role="dnr">Humphredi
+                        Haggat</persName> Convictoris Anno<gap/>
+                     <date calendar="Gregorian" when="1620">MDCXX<hi rend="superscript"
```

Comment:
the date element is unnecessary since the information is already included as an attribute on the acquisition element.